Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"test\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision f224ae98bedff5fc341beba3d083d154bcec1aff)
+++ b/main.py	(date 1622840913322)
@@ -1,1 +1,1 @@
-print("test")
\ No newline at end of file
+print("hello world")
\ No newline at end of file
